src/js/components/molecules/flyout-section/flyout-section.vue.html

```diff
@@ -6,7 +6,7 @@
         :aria-expanded="!isCompact || isExpanded">
 
         <li class="flyout__list-item"  v-for="(item, index) in items" v-if="index < maxVisible">
-            <content-link ref="firstFocusableLinks" @click.native="$emit('close')" :url="item.url" class="flyout__link link" :label="item.label"></content-link>
+            <content-link ref="firstFocusableLinks" @click.native="$emit('content-link-click')" :url="item.url" class="flyout__link link" :label="item.label"></content-link>
```

Comment:
Should content-link-click be more descriptive?

Reply:
not sure. i chose this path to make the event generic and let the parent decide what to do with it. in this case the parent will close the mobile nav, but what the parent does could be different.